code/modules/mob/living/carbon/human/human.dm

```diff
@@ -941,12 +941,18 @@
 	return number
 
 /mob/living/carbon/human/check_ear_prot()
+	if(!src.can_hear())
+		return HEARING_PROTECTION_TOTAL
+	if(l_ear && istype(l_ear, /obj/item/clothing/ears/earmuffs))
```

Comment:
Can simplify this one and the one on line 948 to just an istype. `istype(null)` will return FALSE